cellrank/kernels/_velocity_kernel.py

```diff
@@ -31,13 +31,18 @@ class VelocityKernel(ConnectivityMixin, BidirectionalKernel):
     ----------
     %(adata)s
     %(backward)s
+    attr
+        Attribute of `anndata.AnnData` to be read from. Should be `anndata.AnnData.layers` or `anndata.AnnData.obsm`.
```

Comment:
Would just just use:
```
Attribute of :class:`~anndata.AnnData` to read from.
```
as this should be pretty clear from the docs of `{x,v}key` where we read from.